pkg/options/options.go

```diff
@@ -1,17 +1,26 @@
 package options
 
-type TestOptionsContainer struct {
-	Options TestOptions `yaml:"options"`
-}
+import (
+	"crypto/rand"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"path/filepath"
+
+	libgocmd "github.com/open-cluster-management/library-e2e-go/pkg/cmd"
+
+	"gopkg.in/yaml.v1"
+	"k8s.io/klog"
+)
 
 // Define options available for Tests to consume
-type TestOptions struct {
+type TestOptionsT struct {
 	Hub              Hub             `yaml:"hub,omitempty"`
 	ManagedClusters  ManagedClusters `yaml:"managedClusters,omitempty"`
 	IdentityProvider int             `yaml:"identityProvider,omitempty"`
 	Connection       CloudConnection `yaml:"cloudConnection,omitempty"`
 	Headless         string          `yaml:"headless,omitempty"`
-	OwnerPrefix      string          `yaml:"ownerPrefix,omitempty"`
```

Comment:
This is a nice feature to have in order to know who is creating the namespace and deployment in AWS/GCP/Azure.  Please consider keeping

Reply:
This was move as a parameter of the end-to-end test, not anymore in option but I can deal with both if needed.